server/app/forms/admin/BulkStatusUpdateForm.java

```diff
@@ -5,38 +5,37 @@
 
 /** A form for selecting multiple applications to be considered for a status update. */
 public final class BulkStatusUpdateForm {
-  private List<String> applicationsIds;
+  private List<Long> applicationsIds;
   private String statusText;
-  private Boolean maybeSendEmail;
+  private Boolean shouldSendEmail;
```

Comment:
Question: should this be primitive `boolean` with lowercase "b" instead of Class `Boolean`?

IIRC, a Java primitive is "pass by value". A class is "pass by reference", which would allow code outside this class to change `shouldSendEmail`'s value (and therefore introduce subtle bugs)